src/test/java/hudson/plugins/git/GitChangeSetEuroTest.java

```diff
@@ -60,7 +60,7 @@ public void createEuroChangeSet() {
         gitChangeLog.add("    Tested-by: Jenkins <jenkins@no-mail.com>");
         gitChangeLog.add("    Reviewed-by: Mister Another <mister.another@ericsson.com>");
         gitChangeLog.add("");
-        changeSet = new GitChangeSet(gitChangeLog, useAuthorName);
+        changeSet = new GitChangeSet(gitChangeLog, useAuthorName, false);
```

Comment:
Is there still at least one call in the tests to the prior constructor?

Reply:
Sure, in this test: *GitChangeSetSample*, the three param constructor is mainly for testing, the app should be using the old one (that calls the now one with three params)